clients/pkg/promtail/limit/config.go

```diff
@@ -0,0 +1,17 @@
+package limit
+
+import (
+	"flag"
+)
+
+type Config struct {
+	ReadlineRate        float64 `yaml:"readline_rate" json:"readline_rate"`
+	ReadlineBurst       int     `yaml:"readline_burst" json:"readline_burst"`
+	ReadlineRateEnabled bool    `yaml:"readline_rate_enabled,omitempty"  json:"readline_rate_enabled"`
```

Comment:
I think we should add a config to decided if we want to drop or not the log line. This makes it usable in multiple use case.

A bit like the stage one but global.

Reply:
done,thanks .
![image](https://user-images.githubusercontent.com/9583245/148071780-6e1fc6d7-bc33-4f0c-af7b-95520949f162.png)
![image](https://user-images.githubusercontent.com/9583245/148071856-6149860e-a22a-4157-b2f7-09b4006649b2.png)